### ISSUE TITLE
Hackers Button!

### ISSUE THREAD

**Issue by jedahan** [nycresistor/space-automation#5]
Press the button, hackers plays, shows hours since hackers has played

**Comment by grischard** [comment]
Where would this play Hackers, on the TV? Would it also control/dim the lights?

**Comment by phooky** [comment]
I think @osresearch had this running several years ago, but I don't know what the current status is.

**Comment by jedahan** [comment]
Yeah it would show on tv. I leave it to implementer of how immersive / interruptive they want it to be regarding lights dimming.
